runner.yml.dist

```diff
@@ -10,6 +10,7 @@ drupal:
     user: "root"
     password: ""
   post_install:
+    - "./vendor/bin/drush en oe_paragraphs_media -y"
     - "./vendor/bin/drush en oe_paragraphs_demo -y"
```

Comment:
We can move the code of `oe_paragraphs_media_install()` to `oe_paragraphs_demo` so that we add that paragraph only when we demo the module. This will also help with Behat tests that assumes the paragraph is there.